habitat_baselines/rl/ddppo/policy/resnet_policy.py

```diff
@@ -52,7 +52,7 @@ def __init__(
         policy_config: Config = None,
         aux_loss_config: Optional[Config] = None,
         fuse_keys: Optional[List[str]] = None,
-        **kwargs
+        **kwargs,
```

Comment:
Not sure what was rational to remove it. Idea adding keyword arguments is that for policy initialization through the constructor we will have different set of parameters for different type of policies. These way we can still use same repository of policies, but have different APIs for policies handling different tasks.